pkg/skaffold/docker/parse.go

```diff
@@ -101,10 +101,27 @@ func GetDockerfileDependencies(workspace string, r io.Reader) ([]string, error)
 	// Look for .dockerignore.
 	ignorePath := filepath.Join(workspace, ".dockerignore")
 
-	filteredDeps, err := util.ApplyDockerIgnore(expandedDeps, ignorePath)
+	absWorkspacePath, _ := filepath.Abs(workspace)
```

Comment:
We should handle this error. Abs can return errors in a few cases.

Reply:
Ah missed that one. Pushed a new commit